config/generateProdConfig.test.js

```diff
@@ -0,0 +1,11 @@
+const generateProdConfig = require('./generateProdConfig')
+
+describe('generateProdConfig', () => {
```

Comment:
Is it worth testing the happy path too?

Reply:
It's a bit fidley as it's dependant on env vars.

I could do something like the suggestion in this SO answer:
https://stackoverflow.com/questions/48033841/test-process-env-with-jest